Sources/OpenAPIRuntime/Conversion/Configuration.swift

```diff
@@ -105,17 +123,27 @@ public struct Configuration: Sendable {
     /// The generator to use when creating mutlipart bodies.
     public var multipartBoundaryGenerator: any MultipartBoundaryGenerator
 
+    public var customCoders: [String: any CustomCoder]
```

Comment:
I'm a bit worried about this being too generic and folks assuming it'll override the JSON coders as well (which it won't).

Could you change this to `public var xmlCoder: (any CustomCoder)?` instead? I think it's better to be explicit here.

Reply:
Should we also get rid of `Configuration.customCoder(for:)` method, and use property `xmlCoder` in `Converter.convertXMLToBodyCodable(_:)` and `Converter.convertBodyCodableToXML(_:)` ?

Reply:
Yup exactly. Keep the public protocol `CustomCoder`, but only allow customizing the XML one on the Configuration struct.